eb_hooks.py

```diff
@@ -268,6 +268,22 @@ def pre_configure_hook_wrf_aarch64(self, *args, **kwargs):
         raise EasyBuildError("WRF-specific hook triggered for non-WRF easyconfig?!")
 
 
+def pre_configure_hook_PLUMED_aarch64(self, *args, **kwargs):
+    """
+    pre-configure hook for PLUMED:
+    - remove unsupported --enable-asmjit option on aarch64
+    """
+
+    if self.name == 'PLUMED':
+        if  get_cpu_architecture() == AARCH64:
+            configopts = self.cfg['configopts']
+            regex = re.compile(r'--enable-asmjit')
```

Comment:
Wouldn't it be better to update the `PLUMED` easyconfigs in EasyBuild to only use `--enable-asmjit` on `x86_64` (kinda similar to https://github.com/easybuilders/easybuild-easyconfigs/pull/19000)?

```python
if ARCH == 'x86_64':
    configopts += '--enable-asmjit '
```

according to https://github.com/plumed/plumed2/issues/982, there's even no need to explicitly use `--enable-asmjit`, we should be able to rely on the `configure` script auto-enabling it?

Reply:
Agree with Kenneth, I think we should basically do [this](https://github.com/plumed/plumed2/issues/982#issuecomment-1775474726), i.e. in the EasyConfig:
- for Plumed < 2.8, put `configopts += '--enable-asmjit'` only for `x86_64`
- for Plumed >= 2.8, rely on configure script to auto-enable it